### PR TITLE
Add a display for the current treadmill stats.

### PR DESCRIPTION
**Pull request by caarmen**
Add a `-m` (`--display-mode`) option, with three possible values:
* `plaintext`
* `richtext` (default)
* `json`

The following are example display values:

**plaintext**:
```
Distance: 0.06 km. Duration: 2m. Speed: 2.0 km/h.
Distance: 0.06 km. Duration: 2m. Speed: 2.0 km/h.
Distance: 0.06 km. Duration: 2m. Speed: 2.0 km/h.
Distance: 0.07 km. Duration: 2m. Speed: 2.0 km/h.
Distance: 0.07 km. Duration: 2m. Speed: 2.0 km/h.
Distance: 0.07 km. Duration: 2m. Speed: 2.0 km/h.
```

**richtext**:
```
Distance: 0.03 km
Duration: 1m
Speed: 2.0 km/h
```

**json**:
```
{"distance_m": 80, "duration_s": 161, "speed_kph": 2.0}
{"distance_m": 80, "duration_s": 162, "speed_kph": 2.0}
{"distance_m": 80, "duration_s": 163, "speed_kph": 2.0}
{"distance_m": 90, "duration_s": 164, "speed_kph": 2.0}
{"distance_m": 90, "duration_s": 165, "speed_kph": 2.0}
{"distance_m": 90, "duration_s": 166, "speed_kph": 2.0}
```

### CODE REVIEW

File: walkingpadfitbit/domain/display/factory.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from walkingpadfitbit.domain.display.richtext import RichTextDisplay
+
+
+class DisplayMode(enum.StrEnum):
+    PLAIN_TEXT = "plaintext"
+    RICH_TEXT = "richtext"
+    JSON = "json"
+
+
+def get_display(display_mode: DisplayMode):
```
Comment (caarmen) [inline walkingpadfitbit/domain/display/factory.py:14]: return typehint

File: walkingpadfitbit/domain/display/richtext.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from walkingpadfitbit.domain.display.duration import friendly_duration
+from walkingpadfitbit.domain.entities.event import TreadmillWalkEvent
+
+
+class RichTextDisplay(BaseDisplay):
+    def to_text(
+        self,
+        event: TreadmillWalkEvent,
+    ) -> str:
+        return f"""{chr(27)}[2JDistance: {event.dist_km:.2f} km
```
Comment (caarmen) [inline walkingpadfitbit/domain/display/richtext.py:11]: Need to move to the top left

**Comment by BoD** [comment]
Not sure if possible but if the thing is in stand-by or completely off, can we show:

```
Distance: --
Duration: --
Speed: --
```

(and in json: `{"distance_m": null, "duration_s": null, "speed_kph": null}`)

**Comment by caarmen** [comment]
> Not sure if possible but if the thing is in stand-by or completely off, can we show:
> 
> ```
> Distance: --
> Duration: --
> Speed: --
> ```
> 
> (and in json: `{"distance_m": null, "duration_s": null, "speed_kph": null}`)

This is done :)

**Pull request merged**